src/main/java/org/synyx/urlaubsverwaltung/settings/SettingsServiceImpl.java

```diff
@@ -31,11 +33,22 @@ public Settings save(Settings settings) {
 
     @Override
     public Settings getSettings() {
-        return settingsRepository.findAll().stream().findFirst()
-            .orElseGet(() -> {
-                final Settings savedSettings = settingsRepository.save(new Settings());
-                LOG.info("Saved initial settings {}", savedSettings);
-                return savedSettings;
-            });
+        Iterator<Settings> iterator = settingsRepository.findAll().iterator();
```

Comment:
Müssen wir get splitten in get und insert? Iterator sollte man ggf eine List, ggf neues interface im repo, austauschen. Iterator fühlt sich angestaubt an.

Reply:
ist an das pattern der anderen services angepasst, welche `if (count == 0) {...}` verwenden und daten eintragen.